motoko/invoice-canister/examples/motoko-seller-client/src/declarations/frontend/frontend.did

```diff
@@ -0,0 +1,140 @@
+type BatchId = nat;
```

Comment:
Should src/declarations be in .gitignore rather than these files added here?

Reply:
We don't have it in the `dfx new` gitignore, so I don't think this is the area to make that decision